packages/workflow/package.json

```diff
@@ -51,6 +51,9 @@
     "typescript": "~4.6.0"
   },
   "dependencies": {
+    "@babel/core": "^7.18.13",
+    "@babel/plugin-transform-arrow-functions": "^7.18.6",
+    "@babel/types": "^7.18.13",
```

Comment:
`@babel/types` is pulled by every single babel package implicitely. I don't think we need to explicitely add it here; especially that I don't see any code in here explicitely importing this.